all/entrypoint.sh

```diff
@@ -170,8 +170,8 @@ PROTO_INCLUDE="-I /usr/include/ \
 
 if [ ! -z $PROTO_DIR ]; then
     PROTO_INCLUDE="$PROTO_INCLUDE -I $PROTO_DIR"
-    PROTO_FILES=(`find ${PROTO_DIR} -maxdepth 1 -name "*.proto"`)
-else 
+    PROTO_FILES=(`find ${PROTO_DIR} -name "*.proto"`)
```

Comment:
I tested this out locally and it worked just fine for my use-case where I have nested `.proto` files.